yamahayxc/plugin.yaml

```diff
@@ -61,16 +63,116 @@ plugin:
 #    documentation: https://github.com/smarthomeNG/smarthome/wiki/CLI-Plugin        # url of documentation (wiki) page
     support: https://knx-user-forum.de/forum/supportforen/smarthome-py/1174064-plugin-yamaha-musiccast-geräte-neuere-generation
 
-    version: 1.0.5                 # Plugin version
-    sh_minversion: 1.3             # minimum shNG version to use this plugin
+    version: 1.0.6                 # Plugin version
+    sh_minversion: 1.7             # minimum shNG version to use this plugin
```

Comment:
Which breaking change leads to incompatibility to older SHNG versions?

Reply:
No breaking change as far as I know. Nested structs only work with 1.7, structs earlier. Base code is not changed in respect to core libraries. Reverted minversion to 1.3